conf/evolutions/default/105.sql

```diff
@@ -0,0 +1,22 @@
+# --- !Ups
+
+
+ALTER TABLE `patient_encounters`
+  ADD COLUMN `isDeleted` DATETIME NULL DEFAULT NULL AFTER `date_of_diabetes_screen`;
+ALTER TABLE `patient_encounters`
+  ADD COLUMN `deleted_by_user_id` INT(11) NULL DEFAULT NULL  AFTER `isDeleted`;
```

Comment:
user_id needs to have a foreign key constraint to the user's table so the database enforces restrictions on Ids that can be put in this column (they have to exist in the user's table to add them)

You can see an example here:
https://github.com/FEMR/femr/blob/master/conf/evolutions/default/104.sql#L6